ios/Runner/Info.plist

```diff
@@ -60,5 +60,7 @@
 	<false/>
 	<key>CADisableMinimumFrameDurationOnPhone</key>
 	<true/>
+	<key>UIApplicationSupportsIndirectInputEvents</key>
+	<true/>
```

Comment:
Please double-check to make sure we actually need this. Check if removing this value keeps the existing functionality intact.

Reply:
After running `flutter run --flavor development --target lib/main_development.dart` this command above line getting added again automatically, seems this is required by IDF to run the project.